src/lib/y2storage/region.rb

```diff
@@ -0,0 +1,60 @@
+require "y2storage/storage_class_wrapper"
+
+module Y2Storage
+  # Class representing a certain space in a block device.
+  #
+  # Basically a start/length pair with a block size.
+  #
+  # This is a wrapper for Storage::Region, but has a fundamental difference.
+  # Y2Storage::Region.new receives the Storage::Region object to wrap as the
+  # only parameter (as usual with Storage wrappers). To create an instance from
+  # scratch (generating the corresponding Storage::Region in the process), use
+  # Y2Storage::Region.create, that gets the same arguments than
+  # Storage::Region.new.
+  # @see StorageClassWrapper
+  class Region
+    include StorageClassWrapper
+    wrap_class Storage::Region
+
+    # @raise [Exception] when comparing Regions with different block_sizes
+    storage_forward :==
+    # @raise [Exception] when comparing Regions with different block_sizes
+    storage_forward :!=
+    # @raise [Exception] when comparing Regions with different block_sizes
+    storage_forward :>
+    # @raise [Exception] when comparing Regions with different block_sizes
+    storage_forward :<
```

Comment:
in general, I think it will be much nicer to simply add `def <=>` and include Comparable, to have full set of methods.